pkg/updater/updater_test.go

```diff
@@ -3128,6 +3128,62 @@ func TestAppendColumn(t *testing.T) {
 	}
 }
 
+func TestDynamicEmails(t *testing.T) {
+	columnWithEmails := statepb.Column{Build: "columnWithEmail", Started: 100 - float64(0), EmailAddresses: []string{"email1@", "email2@"}}
+	anotherColumnWithEmails := statepb.Column{Build: "anotherColumnWithEmails", Started: 100 - float64(1), EmailAddresses: []string{"email3@", "email2@"}}
+	columnWithoutEmails := statepb.Column{Build: "columnWithoutEmails", Started: 100 - float64(2)}
+	cases := []struct {
```

Comment:
Think 1 more test case would be a good idea (2 columns have dynamic emails, and the correct emails are chosen).